ui/pages/Search/VariantSearch.jsx

```diff
@@ -5,7 +5,7 @@ import { Grid } from 'semantic-ui-react'
 
 import { Error404 } from 'shared/components/page/Errors'
 import VariantSearchForm, { NoEditProjectsVariantSearchForm } from './components/VariantSearchForm'
-import VariantSearchResults from './components/VariantSearchResults'
+import VariantSearchResults from './components/VariantSearchResultsNew'
```

Comment:
I renamed this file because otherwise github was not doing a good job detecting the diff, I am going to put up another cleanup PR branched off of this one. to change it back